presto-main/src/main/java/com/facebook/presto/operator/aggregation/AggregationMetadata.java

```diff
@@ -252,4 +251,33 @@ static ParameterType inputChannelParameterType(boolean isNullable, boolean isBlo
             }
         }
     }
+
+    public static class AccumulatorStateInfo
```

Comment:
I am thinking to remove `stateInterface`, and call this class `AccumulatorStateSerde`

Reply:
I agree that `XInfo` is not a good name here. What about `AccumulatorStateDescriptor`?